apps/arweave/src/ar_chunk_storage.erl

```diff
@@ -871,6 +873,29 @@ is_offset_valid(Byte, BucketStart, ChunkOffset) ->
 	Delta = Byte - (BucketStart + ChunkOffset rem ?DATA_CHUNK_SIZE),
 	Delta >= 0 andalso Delta < ?DATA_CHUNK_SIZE.
 
+filter_by_sync_record(Chunks, _Byte, _Start, _ChunkFileStart, _StoreID, 1) ->
+	Chunks;
```

Comment:
```suggestion
	%% The code paths which query a single chunk have already implicitly checked that
	%% the chunk belongs to the sync_record. E.g. ar_chunk_storage:get/2
	Chunks;
```

Does this work for a comment to explain?

What about ar_chunk_storage:get/3? It looks like we call it directly from `ar_entropy_storage` now: 
- https://github.com/ArweaveTeam/arweave/blob/master/apps/arweave/src/ar_entropy_storage.erl#L225
- https://github.com/ArweaveTeam/arweave/blob/master/apps/arweave/src/ar_entropy_storage.erl#L326

 I think both of those locations already check that it exists in the sync_record, but just want to double check with you.